common/src/main/scala/explore/Icons.scala

```diff
@@ -369,9 +369,13 @@ object Icons {
   val faLockKeyhole: FAIcon = js.native
 
   @js.native
-  @JSImport("@fortawesome/pro-solid-svg-icons", "faHourglassClock")
+  @JSImport("@fortawesome/pro-regular-svg-icons", "faHourglassClock")
   val faHourglassClock: FAIcon = js.native
 
+  @js.native
+  @JSImport("@fortawesome/pro-regular-svg-icons", "faPlanetRinged")
+  val faPlanetRinged: FAIcon = js.native
```

Comment:
Andy requested this for the non-sidereals, so I added it now. Changed the target icons from `solid` to `regular` because the hourglass and ringed planet looked better that way.